ea_detthresh.m

```diff
@@ -1,9 +1,22 @@
 function thresh=ea_detthresh(atlases,atlas,img)
-% function that will return the threshold according to user settings for a
+% Function that will return the threshold according to user settings for a
 % specific atlas.
-% __________________________________________________________________________________
-% Copyright (C) 2015 Charite University Medicine Berlin, Movement Disorders Unit
-% Andreas Horn
+%
+% USAGE:
+%
+%    thresh = ea_detthresh(atlases,atlas,img)
+%
+% INPUTS:
+%    atlases:
+%    atlas:
+%    img:
+%
+% OUTPUT:
+%    thresh:
+%
+% .. AUTHOR:
+%       - Andreas Horn, Original file
```

Comment:
Andreas Horn cited as the only author of the original file, since the original file only stated Andreas Horn as author. Please tell me if i should only include Andreas in the functions that are signed by him only for future functions as well or i should change this.